helm/templates/deployment.yaml

```diff
@@ -61,6 +61,8 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        - name: AWS_ACCOUNT_ID
```

Comment:
This should be removed.

Reply:
fixed 
generated code with codegen commit 4b30ff557